flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -1504,6 +1506,23 @@
                     .runtimeDeferred()
                     .build();
 
+    public static final BuiltInFunctionDefinition JSON_VALUE =
+            BuiltInFunctionDefinition.newBuilder()
+                    .name("JSON_VALUE")
+                    .kind(SCALAR)
+                    .inputTypeStrategy(
+                            sequence(
+                                    logical(LogicalTypeFamily.CHARACTER_STRING),
+                                    and(logical(LogicalTypeFamily.CHARACTER_STRING), LITERAL),
+                                    TYPE_LITERAL,
+                                    symbol(JsonValueOnEmptyOrError.class),
+                                    ANY,
+                                    symbol(JsonValueOnEmptyOrError.class),
+                                    ANY))
+                    .outputTypeStrategy(SpecificTypeStrategies.JSON_VALUE)
```

Comment:
Ah, we said this can now also be replaced by an existing strategy already. Will do so with the next cycle of updates.